### ISSUE TITLE
Does not work on windows

### ISSUE THREAD

**Issue by sandorex** [vonshednob/pter#4]
I installed it using pip and install `PyQt5` and all i get when running is
```
PS C:\Users\Sandorex> pter
Neither PyQt5 nor curses are installed.
```

**Comment by vonshednob** [comment]
Thanks for reporting this!

The default interface is based on curses and is known to not work that great on Windows in the first place, even if you install [windows-curses](https://pypi.org/project/windows-curses/) from pypi. So Qt5 is a good choice.

Could you tell me what happens when you attempt to import `PyQt5` from your python prompt?

**Comment by sandorex** [comment]
Ah this is my bad, my mind did not see the section about PyQt, `qpter` does work!
Sorry for the trouble

**Comment by vonshednob** [comment]
No problem, glad it works. Maybe I should be more explicit in the README then. :thinking:

**Comment by sandorex** [comment]
> 
> 
> Maybe I should be more explicit in the README then. 🤔

Maybe just placing the command for launching with QT just below the TUI one would prevent someone from missing it like i did

<br><br>

To launch pter you have to tell it where your todo.txt file is:

```shell
pter ~/todo.txt

# GUI mode using QT (requires PyQt5)
qpter ~/todo.txt
```

...

**Comment by vonshednob** [comment]
Fixed in  9def513c836f6789aa0b7b1881175520b12b845f
